tests/Browser/ApiSettingsTest.php

```diff
@@ -5,9 +5,9 @@
 use Illuminate\Foundation\Testing\DatabaseMigrations;
 use Laravel\Dusk\Browser;
 use Tests\DuskTestCase;
-use Tests\Browser\Pages\HomePage;
+use Tests\Browser\Pages\StorePage;
 
-class ExampleTest extends DuskTestCase
+class ApiSettingsTest extends DuskTestCase
 {
     /**
      * A basic browser test example.
```

Comment:
Ah, here's the browser test that I was missing in one of my comments below. Maybe this is not so basic anymore, since it actually tests the un-authenticated view? And maybe it's worth adding a second one for the authenticated user, with the expecation to see a "Log out" link? Or is that too complicated to manage in a browser test?

Reply:
yeah, these are the browser tests I was referring to yesterday, see #70 for more info